app/components/point-input.jsx

```diff
@@ -0,0 +1,191 @@
+                    onFocus={() => setIsSubjFocused(true)}
+                    onBlur={() => setIsSubjFocused(false)}
+                    className={subjCharCount > maxCharCount ? classes.subject + ' ' + classes.errorInput: classes.subject}>
+
+                </input>
+                {(isSubjFocused || subjCharCount > maxCharCount) && (<span
+                    className={classes.wordCount}
+                >
+                    {subjCharCount} / {maxCharCount}
+                </span>)}
+
+                <textarea
+                    ref={textareaRef}
+                    placeholder="Description"
+                    value={description}
+                    onChange={(e) => handleDescriptionChange(e.target.value)}
+                    onBlur={handleDescriptionBlur}
+                    onFocus={() => setIsDescFocused(true)}
+                    className={descWordCount > maxWordCount ? classes.description + ' ' + classes.errorInput: classes.description}>
+                </textarea>
+                {(isDescFocused || descWordCount > maxWordCount) && (<span
+                    className={classes.wordCount}
+                >{descWordCount} / {maxWordCount}</span>)}
+            </div>
+        </>
+    )
+};
+
+
+const useStyles = createUseStyles(theme => ({
+
+    container: {
+        display: 'flex',
+        flexDirection: 'column',
+        top: '4.0625rem',
+        left: '2.75rem',
+        gap: '0.625rem',
+        width: '100%',
+        height: '13.125rem',
+    },
+    subject: {
+        ...sharedInputStyle(theme),
+        padding: '0.9375rem',
+        height: '2.8125rem',
+        '&::placeholder': {
+            ...sharedPlaceholderStyle(theme),
+        },
+        '&[type="text"]': {
+            border: `0.0625rem solid ${theme.colors.borderGray}`,
+            color: theme.colors.title,
+            fontSize: '1rem',
+            lineHeight: '1.5rem'
+        },
+        '&[type="text"]:hover': {
+            ...sharedHoverStyle(theme),
+        },
+    },
+    description: {
+        ...sharedInputStyle(theme),
+        resize: 'none',
+        width: '100%',
+        padding: '0.9375rem 0.9375rem 1.25rem 0.9375rem',
+        '&::placeholder': {
+            ...sharedPlaceholderStyle(theme),
+        },
+        '&:hover': {
+            ...sharedHoverStyle(theme),
+        },
+    },
+    wordCount: {
+        textAlign: 'right',
+        fontFamily: theme.font.fontFamily,
+        fontSize: '0.875rem',
+        fontStyle: 'normal',
+        fontWeight: '300',
+        lineHeight: '1.125rem'
+    },
+
+    errorInput: {
+        borderRadius: '0.25rem',
+        ...sharedErrorStyle(theme),
+        '&:hover': {
+            ...sharedErrorStyle(theme)
+        },
+        '&[type="text"]': {
+            ...sharedErrorStyle(theme)
+        },
+        '&[type="text"]:hover': {
+            ...sharedErrorStyle(theme)
+        }
+
+    }
+}))
+
+const sharedPlaceholderStyle = theme => ({
+    width: '31.6875rem',
```

Comment:
let the parent determine the width. You may need to use padding or something to fit the placeholder in the container.